content/millennium/r4/bulk-data/group-export.md

```diff
@@ -0,0 +1,301 @@
+
+By default, the Group Export operation exports resources required for [United States Core Data for Interoperability (USCDI)](https://www.healthit.gov/isa/united-states-core-data-interoperability-uscdi). For a full list of resources see the [US Core Profiles](https://hl7.org/fhir/us/core/#us-core-profiles). When the _type parameter is used it returns only the FHIR resources specified. The _type parameter should be used to filter the resources required for a specific use case. Limiting the number of requested resources in the _type parameter is recommended as it can decrease response times.
+
+Client application developers will need to work with healthcare organizations to be granted access to the Group $export operation and to be provided with Groups to be exported. Organizations can manage Groups within [Ignite Management Tooling](https://cernercentral.com/ignite-management).
+
+_Notes_
+
+* Authentication is required, open endpoints do not support bulk export workflows
+* Cerner supports v1.0.1 of the [FHIR Bulk Data Access (FLATFHIR) Implementation Guide](https://hl7.org/fhir/uv/bulkdata/STU1.0.1/)
+* Bulk data runs against an organization's database and utilizes the same services running other applications, so it is important to consider performance. Bulk data exports large amounts of data for large groups of patients, which takes more time to complete the larger the data set or patient population.
+
+### When to use bulk data
+
+The Bulk Data Export API enables connecting applications to mine large amounts of data; however, its inherent flexibility also presents challenges. When implementing an interface between a Cerner Millennium domain and a bulk data app, it is very important that the queries inherent in the bulk data request capture the data necessary for the app – but little or no more. Furthermore, export files should be deleted as soon as they are no longer needed.
+
+There are several factors that affect the amount of time it takes to complete a Bulk API export:
+
+* The number of patients to whom the request applies (that is, the Group)
+* The number of resource types requested
+* The time range of activity being requested
+* Inherent latency due to network issues and effects from concurrent requests to the system
+
+#### Example use cases
+
+* Creating an internal clinical data warehouse for study purposes
+* Consolidation of claims data for comprehensive view
+* Machine learning applications obtaining training data from EHR
+* Integration of population health system with an EHR system
+* Transferring records between EHRs
+* Data transfer to payer database for care quality assessment
+* Submission to reportable disease (or other) registries
+
+### Best Practices
+
+* Limit group sizes to around a thousand patients or fewer.
```

Comment:
Place holder - we need to decide what we will recommend on group size if we don't have our abilities lab results back by the time we need to publish.

Reply:
Resolving this comment in favor of this one: https://github.com/cerner/fhir.cerner.com/pull/899#discussion_r978700516